docs/step-by-step/kubeapps-on-tkg/step-1.md

```diff
@@ -0,0 +1,127 @@
+3. Apply this file to the cluster:
+
+```bash
+kubectl apply -f pinniped-0.8-tmc.yaml
+```
+
+4. Confirm that Pinniped has been installed and the image is correct:
+
+```bash
+kubectl get deploy/pinniped-concierge-0-8-0 -n pinniped-concierge-0-8-0 -oyaml | grep image
+```
+
+At this point, a TKG cluster with a compatible Pinniped version is up and running.
+
+### Step 1.2: Configure an OIDC Provider
+
+The next step is to configure an OIDC provider and then configure Pinniped to trust the OIDC provider. The OIDC provider is responsible for authenticating users in the TKG cluster and the Kubeapps dashboard. The steps below use the [VMware Cloud Services Platform (CSP)](https://console.cloud.vmware.com/) as an example; however, a similar process applies to any other OIDC-compliant provider, including Google Cloud, Azure Active Directory, Dex, Okta, and others.
+
+#### Create an OAuth2 Application
+
+Begin by creating an OAuth2 application to retrieve the information required by Pinniped and, later on, Kubeapps. Follow the steps below:
+
+> **NOTE**: You must have _Developer_ access in the organization to perform these steps.
+
+1. Navigate to the CSP Console at [https://console.cloud.vmware.com](https://console.cloud.vmware.com/).
+2. Click the drop-down menu in the top-right corner.
+3. Under the _Organization_ settings, click _View Organization_.
+
+   ![View organization](./img/csp-menu-organization.png)
+
+4. Click the _OAuth Apps_ tab.
+
+   ![OAuth Apps tab](./img/csp-oauth-initial.png)
+
+5. Select _Web app_ and click the _Continue_ button.
+
+   ![OAuth Apps tab](./img/csp-oauth-new.png)
+
+6. Enter a name and description for the OAuth app. For the moment, enter the value `https://localhost/oauth2/callback` in the _Redirect URIs_ field (this will be updated after Kubeapps is installed).
+
+   ![Add name and description](./img/csp-oauth-new-details-general.png)
+
+7. Leave the rest of the options at their default values.
+8. Tick the _OpenID_ checkbox and click the _Create_ button.
+
+   ![OpenID and create](./img/csp-oauth-new-details-scopes.png)
+
+The CSP Console displays a success screen with an auto-generated application ID and secret. Click the _Download JSON_ link to download these values.
+
+![Retrieve app id and secret](./img/csp-oauth-new-secrets.png)
+
+> **NOTE**: Store this file carefully as it contains important credentials which will be needed when configuring Pinniped and Kubeapps.
+
+> **TIP**: Any OIDC-compliant provider should expose a `.well-known/openid-configuration` ([example](https://console.cloud.vmware.com/csp/gateway/am/api/.well-known/openid-configuration)) endpoint where you can find other useful and required information. This endpoint allows using the base URL to discover the rest of the URLs (`authorization`, `token`, `end session`, `jwks` and `issuer`) automatically. For CSP, the endpoint is `https://console-stg.cloud.vmware.com/csp/gateway/am/api`.
```

Comment:
Earlier the doc directs to production csp, but this URI is for staging?

Reply:
You're right! Thank you! I've checked other references as well.